src/Binance/USDMFutures/API/LongShortRatio.jl

```diff
@@ -0,0 +1,96 @@
+module LongShortRatio
+
+export LongShortRatioQuery,
+    LongShortRatioData,
+    long_short_ratio
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Binance
+using CryptoAPIs: Maybe, APIsRequest
+
+@enum Period m5 m15 m30 h1 h2 h4 h6 h12 d1
```

Comment:
I think that you should to rename all enums with the name `Period` to `TimePeriod` because this will cause confusion with the similar [type](https://docs.julialang.org/en/v1/stdlib/Dates/#Dates.Period) name from the base `Dates` package

Reply:
Maybe we should stick to one name?

https://github.com/bhftbootcamp/CryptoAPIs.jl/blob/38858b6fe4345925da0a45a59fc95f2440b02831/src/Binance/Spot/API/Candle.jl#L13

Reply:
In general, we can name all such enums as `TimeInterval` if there are no problems between periods and intervals in the future
I just don’t quite understand why Binance introduces periods and intervals for these ideologically similar parameters

Reply:
@artememelin, ¯\_(ツ)_/¯